src/raftstore/store/store.rs

```diff
@@ -121,6 +121,8 @@ pub struct Store<T, C: 'static> {
     region_written_bytes: LocalHistogram,
     region_written_keys: LocalHistogram,
     lock_cf_written_bytes: u64,
+
+    bytes_written: u64,
```

Comment:
do we need to extract a store metric structure to save those bytes and keys fields?